src/NotepadNext/dialogs/MainWindow.cpp

```diff
@@ -1252,7 +1252,7 @@ void MainWindow::activateEditor(ScintillaNext *editor)
 void MainWindow::setLanguage(ScintillaNext *editor, const QString &languageName)
 {
     qInfo(Q_FUNC_INFO);
-    qInfo("%s", qUtf8Printable("Language Name: " + languageName));
+    qInfo("%s", qUtf8Printable(tr("Language Name: ") + languageName));
```

Comment:
This is just for debugging and not needed to be translated.

Reply:
done